translations/it.php

```diff
@@ -334,9 +343,11 @@
 $_MODULE['<{ps_checkout}prestashop>translations_230a08d54d32844a29f3b5e11cd67946'] = 'Attivazione dei campi alternativi della carta di credito';
 $_MODULE['<{ps_checkout}prestashop>translations_6f266e3508d20d99c079e0130c679208'] = 'Campi della carta di credito PayPal';
 $_MODULE['<{ps_checkout}prestashop>translations_8acc0bc61a5f75b13cea7e27bbcd51af'] = 'È possibile scegliere il tipo di campi della carta di credito solo se la carta di credito è attivata nella scheda " Personalizza esperienza di pagamento".';
+$_MODULE['<{ps_checkout}prestashop>translations_052a5961a6b60e7acd69643f309f47fd'] = 'I campi carta di credito alternativi di PayPal non sono disponibili perché i campi carta integrati che forniscono la migliore esperienza di acquisto sono già abilitati.';
 $_MODULE['<{ps_checkout}prestashop>translations_19146f30e246bba7235f1f052711fe74'] = 'Qual è la differenza tra i campi carta di credito integrati e i campi della carta di credito PayPal?';
 $_MODULE['<{ps_checkout}prestashop>translations_ff8313e7d0a38ececa23e28fd77421de'] = 'Quando usare i campi della carta di credito PayPal?';
 $_MODULE['<{ps_checkout}prestashop>translations_4cb668c4f89af857decf3d6ce310cd77'] = 'I campi carta di credito integrati offrono la migliore esperienza di pagamento possibile su PrestaShop. Ben integrati nel processo di pagamento, privi di marca, con il minor numero di campi possibile e le tariffe più basse (vedi i campi su';
+$_MODULE['<{ps_checkout}prestashop>translations_3a8e02212752fecc2b5eb72111cdc40d'] = '): ti consigliamo di dare la preferenza a loro. Il tuo account PayPal deve essere stato completamente approvato per trarne vantaggio. Puoi vedere lo stato di approvazione del tuo account PayPal nel file';
```

Comment:
There is a blank space for this translation on en.php and fr.php files.
Also, it doesn't exists on other languages, maybe it's the same thing for some other translations. @Darmona

Reply:
Hi, yes you are right. :)
But I think there is an error about the fr.php language.
I mean, I think, this sentence is the second part of the first part indicated on previously line (349) so, in that line there is no white space after the opening brakets. This is the reason there is no white space here.

Reply:
Hello !
In fact, I think there is no errors on english and french files. This is a syntax convention. By example, on your comment you added blank spaces before and after opening and closing brackets: "line (349) so". :)

Reply:
Sorry now I understand. My apologies.
Thank you :)